### PR TITLE
Hide implementation details of Pydoxine generation.

### PR DESCRIPTION
**Pull request by eile**
This is a typical case of making things complex: Why would the library
care that the files are absolute? Why do I need to specify the
(undocumented) include path separately? Should imo not have passed CR.

### CODE REVIEW

File: brain/python/CMakeLists.txt [hunk 1]
```diff
@@ -22,18 +22,12 @@ set(BRAIN_PYTHON_SOURCES
...
-foreach(__source ${BRAIN_PYTHON_SOURCES})
-  list(APPEND __docstrings_sources ${CMAKE_CURRENT_SOURCE_DIR}/${__source})
-endforeach()
+docstrings(BRAIN_PYTHON_SOURCES BRAIN_PUBLIC_HEADERS
+  ${PROJECT_BINARY_DIR}/include)
 
-set(DOCSTRINGS_INCLUDE_PATH ${PROJECT_BINARY_DIR}/include)
-docstrings(__docstrings_sources __docstrings_headers)
+if(TARGET Brion-docstrings)
+  add_definitions(-DBRION_HAS_DOCSTRINGS)
```
Comment (tribal-tec) [inline brain/python/CMakeLists.txt]: target_compile_definitions(Brain PRIVATE BRION_HAS_DOCSTRINGS)

Comment (hernando) [inline brain/python/CMakeLists.txt]: It should be brain_python then, not Brain

Comment (tribal-tec) [inline brain/python/CMakeLists.txt]: And BRAIN_HAS_DOCSTRINGS, not BRION_

Comment (eile) [inline brain/python/CMakeLists.txt]: done

**Comment by hernando** [comment]
reset this please

**Comment by hernando** [comment]
retest this please

**Pull request merged**